code/clue.js

```diff
@@ -1,59 +1,361 @@
+const poison = {
+  name: 'Poison',
+  id: 'poison',
+  weight: 3,
+  color: '#5656b3',
+  article: '',
+  increasedChanceOfDraw: 7,
+  likelyFoundIn: 'Study'
+};
+
+const axe = {
+  name: 'Axe',
+  id: 'axe',
+  weight: 6,
+  color: '#fff',
+  article: 'an',
+  increasedChanceOfDraw: 10,
+  likelyFoundIn: 'Billiard Room'
+};
+
+const bat = {
+  name: 'Bat',
+  id: 'bat',
+  weight: 2,
+  color: '#fff',
+  article: 'a',
+  increasedChanceOfDraw: 20,
+  likelyFoundIn: 'Lounge'
+};
+
+const trophy = {
+  name: 'Trophy',
+  id: 'trophy',
+  weight: 2,
+  color: '#539853',
+  article: 'a',
+  increasedChanceOfDraw: 8,
+  likelyFoundIn: 'Theater'
+};
+
+const pistol = {
+  name: 'Pistol',
+  id: 'pistol',
+  weight: 2,
+  color: '#fbd460',
+  article: 'a',
+  increasedChanceOfDraw: 25,
+  likelyFoundIn: 'Living Room'
+};
 
 const suspects = [
   mrGreen,
+  clMustard,
+  mrsPeacock,
+  prPlum,
+  msScarlet,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candleStick,
+  dumbBell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
+];
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+const mystery = {
+  killer: null,
+  weapon: null,
+  room: null
+};
+
+const pickKiller = (loadID) => {
+  if (gameStarted && !killerPicked) {
+    document.getElementById(loadID.id).style.opacity = 1;
+    setTimeout(function () {
+      document.getElementById(loadID.id).style.opacity = 0;
+      mystery.killer = randomSelector(suspects);
+      document.getElementById('killerName').style.background = mystery.killer.color;
+      document.getElementById('killerImage').src = mystery.killer.image;
+      document.getElementById('killerName').innerHTML = (`${mystery.killer.firstName} ${mystery.killer.lastName}`);
+      const weaponID = mystery.killer.favoriteWeapon;
+      const weaponName = weapons.find(weapon => weapon.id === weaponID).name;
+      document.getElementById('killerFavoriteWeapon').innerHTML = (`Preferred weapon: ${weaponName}`);
+      document.getElementById('killerAge').innerHTML = (`Age: ${mystery.killer.age}`);
+      document.getElementById('killerOccupation').innerHTML = (`Occupation: ${mystery.killer.occupation}`);
+      document.getElementById('killerDescription').innerHTML = (`Description: ${mystery.killer.description}`);
+      killerPicked = true;
```

Comment:
I'm interested to know why you chose to create a variable for this rather than relying on the `mystery` data.  

Another alternative is to create a function like
```javascript
const isKillerPicked = () => mystery.killer !== null;
```
Or something similar